projects/valtimo/form-view-model/src/lib/components/form-view-model/form-view-model.component.ts

```diff
@@ -209,4 +207,16 @@ export class FormViewModelComponent implements OnInit {
       }
     });
   }
+
+  private handleChanges() {
+    this.change$.pipe(pairwise()).subscribe(([prevChange, currentChange]) => {
+      console.log(prevChange?.changed?.value, currentChange?.changed?.value);
+      if (prevChange?.changed?.value == undefined || currentChange?.changed?.value == undefined) {
```

Comment:
undefined check is in javascript also done using !== I thought

from chat gpt

!= (Loose Inequality):

This operator checks for inequality between two values after performing type coercion if necessary. Type coercion means that JavaScript will convert one or both values to a common type before making the comparison.
Example:
javascript
Copy code
console.log(5 != '5'); // false, because '5' is coerced to 5 before comparison
console.log(5 != 6);   // true, because 5 is not equal to 6
console.log(null != undefined); // false, because null and undefined are considered equal with loose comparison
!== (Strict Inequality):

This operator checks for inequality between two values without performing type coercion. Both the value and the type must be different for !== to return true.
Example:
javascript
Copy code
console.log(5 !== '5'); // true, because the types are different (number vs string)
console.log(5 !== 6);   // true, because 5 is not equal to 6
console.log(null !== undefined); // true, because null and undefined are different types

Reply:
I think you can combine it then:

      if (prevChange?.changed?.value !== currentChange?.changed?.value) {
        this.updateViewModel();
      }